jetbrains-rider/tst/software/aws/toolkits/jetbrains/services/lambda/completion/DotNetHandlerCompletionTest.kt

```diff
@@ -36,6 +36,8 @@ class DotNetHandlerCompletionTest : BaseTestWithSolution() {
     }
 
     // TODO this test only works on 2019.2. Which we don't support anymore. Fix the test
+    // TODO: This test is failing due to handlers detection logic. I assume it need to be fixed if test is correct.
```

Comment:
I think the first TODO was wrong, is this still failing?

Reply:
Yes. This is because of the lambda detection logic. The main question is where is the problem: in test or in backend logic.

Reply:
@abrooksv, I think we can handle this as a separate issue if there are any changes required from any side. What do you think? I can do it in this PR, but need to figure out what to fix here. Could you please help with this?

Reply:
Ya, we can follow up on this one.